modules/integration/src/test/scala/scala/cli/integration/InstallHomeTests.scala

```diff
@@ -61,6 +61,24 @@ class InstallHomeTests extends munit.FunSuite {
     )
   }
 
+  private def uninstallScalaCli(
+    root: os.Path,
+    binDirPath: os.Path,
+    force: Boolean,
+    skipCache: Boolean
+  ) = {
+    // format: off
+    val cmdUninstall = Seq[os.Shellable](
+      TestUtil.cli, "uninstall",
+      "--binary-name", dummyScalaCliBinName,
+      "--bin-dir", binDirPath
+    ) 
+    ++ (if(force) Seq[os.Shellable]("--force") else Seq.empty)
+    ++ (if(skipCache) Seq[os.Shellable]("--skip-cache") else Seq.empty)
+    // format: on
+    os.proc(cmdUninstall).call(cwd = root)
```

Comment:
NIT We don't have to always resort to always explicitly typing with `os.Shellable` here:
```suggestion
    // format: off
    val cmdUninstall = Seq[os.Shellable](
      TestUtil.cli, "uninstall",
      "--binary-name", dummyScalaCliBinName,
      "--bin-dir", binDirPath
    ) 
    // format: on
    val forceOpts = if (force) Seq("--force") else Seq.empty
    val skipCacheOpts = if (skipCache) Seq("--skip-cache") else Seq.empty
    os.proc(cmdUninstall, forceOpts, skipCacheOpts).call(cwd = root)
```